frontend/src/routes/(authenticated)/admin/[[tab=adminTab]]/AdminTabs.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts" context="module">
+  import type { I18nKey } from '$lib/i18n';
+
+  export const adminTabs = ['projects', 'users'] as const;
+  export type AdminTabId = (typeof adminTabs)[number];
+  const DEFAULT_TAB_I18N: Record<AdminTabId, I18nKey> = {
+    projects: 'admin_dashboard.project_table_title',
+    users: 'admin_dashboard.user_table_title',
+  };
+</script>
+
+<script lang="ts">
+  import t from '$lib/i18n';
+
+  export let activeTab: AdminTabId = 'projects';
+</script>
+
+<div role="tablist" class="hidden admin-tabs:flex tabs tabs-lifted tabs-lg">
+  <div class="tab tab-divider" />
+  {#each adminTabs as tab}
+    {@const isActiveTab = activeTab === tab}
+    <a href={`/admin/${tab}`} data-sveltekit-noscroll data-sveltekit-preload-data="off" role="tab" class:tab-active={isActiveTab} class="tab grow">
+      <h2 class="text-lg flex gap-4 items-center">
+        {#if isActiveTab}
+          <slot>
+            {$t(DEFAULT_TAB_I18N[tab])}
+          </slot>
+        {:else}
+          {$t(DEFAULT_TAB_I18N[tab])}
+        {/if}
+      </h2>
+    </a>
+    <div class="tab tab-divider" />
+  {/each}
+</div>
+
+<h2 class="admin-tabs:hidden text-2xl flex gap-4">
```

Comment:
`admin-tabs:hidden` is really confusing. As I understand `admin-tabs` is a breakpoint, which feels like a clever hack, but it's kinda confusing. Could we either rename it or do it another way that's less confusing? I see in other places you are conditionally applying `admin-tabs:hidden` and I really have a hard time following how that works. I'm not sure what else to call the breakpoint however. Maybe I just need to be more familiar with this style.